drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/cde_export.rb

```diff
@@ -0,0 +1,64 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module HmisExternalApis::AcHmis::Exporters
+  class CdeExport
+    attr_accessor :output
+
+    def initialize(output = StringIO.new)
+      require 'csv'
+      self.output = output
+    end
+
+    def run!
+      Rails.logger.info 'Generating CDE report'
+      write_row(columns)
+      total = cdes.count
+      Rails.logger.info "There are #{total} CDEs to export"
+
+      cdes.each.with_index do |cde, i|
+        Rails.logger.info "Processed #{i} of #{total}" if (i % 100).zero?
+        values = [
+          cde.id,
+          cde.data_element_definition.key,
+          cde.data_element_definition.owner_type.demodulize,
+          cde.owner.id,
+          cde.value,
```

Comment:
Have you tested the case where this returns a Date object? It should write as YYYY-MM-DD ideally. If you copy the `csv_config` from `PathwaysExport`, that should format the dates and the datetimes correctly magically.